newsletter-december-2022.rst

```diff
@@ -0,0 +1,132 @@
+  The tasks are big and small.
+- ✅️ We added a :doc:`readthedocs:unofficial-projects` that will help official maintainers to de-list forks among others.
+- ✅️ We added `an additional auditing feature <https://github.com/readthedocs/readthedocs.org/pull/9607>`_,
+  whereby invitations are added to the Security Log.
+  The feature is available for users of Read the Docs for Business.
+- ✅️ We found, fixed and disclosed a security issue,
+  `XSS: Allow serving of arbitrary HTML files from main domain <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-98pf-gfh3-x3mp>`_.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+
+.. _sphinx6_is_coming:
+
+Sphinx 6 is coming
+------------------
+
+Sphinx 6 is not yet released, but it may be released before the arrival of our next newsletter.
+Here are some of our considerations for getting ready for Sphinx 6:
+
+- Sphinx 6 focuses on reducing the maintenance burden for the future,
+  dropping support of docutils 0.14, 0.15, 0.16 and 0.17.
+  This will make the work of theme and extension maintainers considerably easier,
+  and we may start seeing themes requiring ``Sphinx>=6`` simply because it's less work to support.
+- Sphinx 6 requires docutils 0.18 or 0.19.
+  We are finalizing support for `sphinx-rtd-theme`_ which currently only supports docutils 0.17.
+  These updates are almost finished and are planned to arrive in sphinx-rtd-theme 1.2.0.
+- Sphinx 6 is removing jQuery, but we need it for `sphinx-rtd-theme`_.
+  jQuery is easily added back using the new extension `sphinxcontrib-jquery`_.
+- Sphinx 6 removes all bundled JavaScript libraries.
+  If you depend on any of these,
+  you may have to bundle them yourself or modify your implementation.
+- See also the `Sphinx changelog <https://www.sphinx-doc.org/en/master/changes.html>`_
+
+Since most documentation projects rely on themes and extensions, we recommend that you do not upgrade to Sphinx 6 before you see announcements of Sphinx 6 support in your themes and extensions.
+
+.. _sphinx-rtd-theme: https://sphinx-rtd-theme.readthedocs.io/
+.. _sphinxcontrib-jquery: https://pypi.org/project/sphinxcontrib.jquery/
+
+
+Upcoming features
+-----------------
+
+- We're working on improving our integration with `Material for MkDocs <https://squidfunk.github.io/mkdocs-material/>`_, which is a great theme for `MkDocs <https://www.mkdocs.org/>`_ documentation projects.
+- Many improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- A search redesign to make it nicer across our dashboard and in-doc search experiences. 
+- 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+
+Possible issues
+---------------
```

Comment:
Seems odd not to mention the Sphinx 6 issues here? We should at least link to it.

Reply:
Added!